tests/mmm/test_transformers.py

```diff
@@ -209,6 +210,56 @@ def test_logistic_saturation_min_max_value(self, x, lam):
         assert y_eval.max() <= 1
         assert y_eval.min() >= 0
 
+    def test_scale_preserving_logistic_saturation_m_zero(self):
+        # All values should be zero
+        x = np.ones(shape=(100))
+        y = scale_preserving_logistic_saturation(x=x, m=0.0)
+        np.testing.assert_array_almost_equal(x=np.zeros(shape=(100)), y=y.eval())
+
+    def test_scale_preserving_logistic_saturation_m_max_x(self):
+        # When m == max(x), f(max(x)) is max(x) * 0.76
```

Comment:
does this 0.76 comes from any known close form or analytical expression?

Reply:
It absolutely does, I just didn't do the math yet. I could if it matters :).

Reply:
just a comment will be enough ... at least hinting the computation

Reply:
added a small comment explaining the math